test/Fail/Issue5810.err

```diff
@@ -0,0 +1,3 @@
+Issue5810.agda:8,15-24
+funSort Set LevelUniv is not a valid sort
```

Comment:
Hm, perhaps we could make this error message a bit nicer, e.g. "functions into a type in `LevelUniv` are not allowed".